docs/02.API-REFERENCE.md

```diff
@@ -3768,6 +3768,102 @@ jerry_value_to_bigint (const jerry_value_t value);
 - [jerry_value_is_bigint](#jerry_value_is_bigint)
 - [jerry_get_bigint_digits](#jerry_get_bigint_digits)
 
+## jerry_value_as_integer
+
+**Summary**
+
+Call the ToInteger abstrart operation for the given api value.
```

Comment:
abstrart -> abstract
The comment is incorrect btw. It should only convert numbers and nothing else. Similar cases below.